### ISSUE TITLE
Writing tests

### ISSUE THREAD

**Issue by rohithasrk** [IMGIITRoorkee/django-filemanager#15]
One of the very important requirements of open source packages is being able to test them. Django's tests are good enough as they use unittest framework of python. This is something that needs to be done soon.

**Comment by kartikp1995** [comment]
Go ahead.

**Comment by kartikp1995** [comment]
I think the basic tests are added but more tests need to be added?

**Comment by rohithasrk** [comment]
Yes. The functional tests are to be done. And these are really vital. Working on that.
